roles/swift-common/meta/main.yml

```diff
@@ -22,5 +23,5 @@ dependencies:
     when: openstack_install_method == 'package'
   - role: openstack-distro
     project_name: swift
-    project_packages: "{{ swift.distro.project_packages }}"
```

Comment:
we need project_packages

only lshw should be in dependent pkgs. in default define swift.distro.dependent_packages with one item lshw